### DISCUSSION TITLE
DIVOC feature updates back to Sunbird RC

### DISCUSSION THREAD

**Discussion by sukhpreetssekhon** [Sunbird-RC/community#331] [General]
Are updates being made in DIVOC being checked back in Sunbird RC?
Some of the recently released features can be beneficial for Sunbird RC users as well. 
https://divoc.digit.org/platform/release-notes/divoc-3.5-release-features

@tejash-jl @dileepbapat 

cc - @surendrasinghs @pramodkvarma @rahul101001000

**Comment by parthlawate** [comment]
Good Find @sukhpreetsamagra ! 

The UI changes seem to have a lot of parallels ..  @Renuka-S @tejash-jl @Pratikshakhandagale can we look at the code and see if this can be aligned & brought back to RC ? 

<img width="924" alt="Screenshot 2023-01-31 at 1 44 39 PM" src="https://user-images.githubusercontent.com/673176/215703743-48651665-33cd-4226-8c4b-5c58759a66c8.png">

**Reply by tejash-jl** [reply]
Sure, will get in touch with DIVOC team

**Comment by pramodkvarma** [comment]
Definitely should try to reuse and bring it in.

**Comment by rahul101001000** [comment]
Would love to understand what the process is for upgrades like this across
Sunbird blocks.

Another example is RC inside Sunbird Ed. In UPHRH Ed 5.4.0 was installed
which does not have RC for generating credentials. Ed 5.4.1 has RC. Hoping
the upgrade is smooth, fingers crossed.

Thanks in advance,

Regards
Rahul
